src/odemis/model/_metadata.py

```diff
@@ -88,6 +90,8 @@
 MD_AT_OVV_FULL = "Full overview"
 MD_AT_OVV_TILES = "Built-up overview"
 MD_AT_HISTORY = "History"
+MD_AT_STREAK = "Streak"
```

Comment:
I'd call it more like `MD_AT_TIME_SPEC = "Temporal spectrum"`. "Streak" is a little too much related on how it's done (currently).
Also, you probably can drop the "Streak photocounting" for now.